pkgs/data/fonts/nasin-nanpa/default.nix

```
@@ -0,0 +1,46 @@
+{ lib, stdenvNoCC, fetchurl }:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "nasin-nanpa";
+  version = "2.5.1";
+
+  srcs = [
+    (fetchurl {
+      name = "${pname}.otf";
+      url = "https://github.com/ETBCOR/${pname}/releases/download/n${version}/${pname}-${version}.otf";
+      hash = "sha256-++uOrqFzQ6CB/OPEmBivpjMfAtFk3PSsCNpFBjOtGEg=";
+    })
+    (fetchurl {
+      name = "${pname}-lasina-kin.otf";
+      url = "https://github.com/ETBCOR/${pname}/releases/download/n${version}/${pname}-${version}-lasina-kin.otf";
```

```suggestion
      name = "nasin-nanpa.otf";
      url = "https://github.com/ETBCOR/nasin-nanpa/releases/download/n${version}/nasin-nanpa-${version}.otf";
      hash = "sha256-++uOrqFzQ6CB/OPEmBivpjMfAtFk3PSsCNpFBjOtGEg=";
    })
    (fetchurl {
      name = "nasin-nanpa-lasina-kin.otf";
      url = "https://github.com/ETBCOR/nasin-nanpa/releases/download/n${version}/nasin-nanpa-${version}-lasina-kin.otf";
```
Substituting pname wherever possible just makes the code harder to read